components/esp_lvgl_port/project_include.cmake

```
@@ -0,0 +1,66 @@
+# lvgl_port_create_c_image
+#
+# Create a C array of image for using with LVGL
+function(lvgl_port_create_c_image image_path output_path color_format compression)
+
+    #Get LVGL version
+    idf_component_get_property(lvgl_ver lvgl__lvgl COMPONENT_VERSION)
+    set(LVGL_NAME "lvgl__lvgl")
+    if(lvgl_ver EQUAL "")
+        idf_component_get_property(lvgl_ver lvgl COMPONENT_VERSION)
+    set(LVGL_NAME "lvgl")
+    endif()
+    
+    get_filename_component(image_full_path ${image_path} ABSOLUTE)
+    get_filename_component(output_full_path ${output_path} ABSOLUTE)
+    if(NOT EXISTS ${image_full_path})
+        message(FATAL_ERROR "Input image (${image_full_path}) not exists!")
+    endif()
+    
+    message(STATUS "Generating C array image: ${image_path}")
+    
+    #Create C array image by LVGL version
+    if(lvgl_ver VERSION_LESS "9.0.0")
+        if(${CONFIG_LV_COLOR_16_SWAP} STREQUAL "y")
```

This condition will fail if `CONFIG_LV_COLOR_16_SWAP` is not set
```suggestion
        if(NOT DEFINED ${CONFIG_LV_COLOR_16_SWAP})
            set(CONFIG_LV_COLOR_16_SWAP "n")
        endif()
        if(${CONFIG_LV_COLOR_16_SWAP} STREQUAL "y")
```

Also, please remove trailing whitespaces in this file